### PR TITLE
Replace Dropdown with Menu in Action Group

### LINKED ISSUE #3068: [Housekeeping] `Action Group`: Replace `Dropdown` with `Menu` component
- [X] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the housekeeping request

As the `Menu` component was still in development when `Action Group` was being developed, we used the `Dropdown` component for the "more" menu in `Action Group`.

As the `Menu` component has since been released we should replace the `Dropdown` with the `Menu` component in `Action Group` for improved visibility optimization, i.e. shifting the popout when it cannot fit on either side of the button:

**:x: Current behaviour:**
image

**:white_check_mark: Expected behaviour:**
image

## Have you considered any alternatives?

No.

## Are there any additional context?

We should set the `shift` prop of the floating directive to `true` by default on the `menu` component to ensure the popout doesn't place itself outside the viewport (in the scenario mentioned above ☝🏻)

### PR DESCRIPTION
**Pull request by RasmusKjeldgaard**
## Which issue does this PR close?

This PR closes #3068 

## What is the new behavior?
Menu is now used instead of dropdown, to better handle viewport collisions. 

To do this, I also had to refactor the way we listen for `ionScroll` in the menu component to close the menu when scrolling, to avoid excessive CD cycles, as change detection would fire on every scroll event. Therefore we now add and remove event listeners outside of Angulars zone. 

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Reminders
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Make sure you have updated the cookbook with examples and showcases (for bug fixes, enhancements & new components).

### Review  
- [x] Determine if your changes are a fix, feature or breaking-change, and add the matching label to your PR. If it is tooling, dependency updates or similar, add ignore-for-release.
- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review).
- [ ] Request that the changes are code-reviewed 
- [ ] Request that the changes are [UX reviewed](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#ux-review) (only necessary if your PR introduces visual changes)

When the pull request has been approved it will be merged to `develop` by Team Kirby.

### CODE REVIEW

File: libs/designsystem/action-group/src/action-group.component.ts [hunk 1]
```diff
@@ -17,7 +17,9 @@ import {
   ViewChild,
 } from '@angular/core';
 import { ButtonComponent } from '@kirbydesign/designsystem/button';
-import { DropdownComponent, DropdownModule } from '@kirbydesign/designsystem/dropdown';
+import { DropdownModule } from '@kirbydesign/designsystem/dropdown';
```
Comment (jakobe) [inline libs/designsystem/action-group/src/action-group.component.ts]: Remove 🔥 
```suggestion
```

File: libs/designsystem/action-group/src/action-group.component.ts [hunk 2]
```diff
@@ -31,7 +33,7 @@ type CollapsedAction = { button: HTMLButtonElement; text: string };
 @Component({
   selector: 'kirby-action-group',
   standalone: true,
-  imports: [CommonModule, ButtonComponent, DropdownModule],
+  imports: [CommonModule, ButtonComponent, ItemModule, MenuComponent, DropdownModule],
```
Comment (jakobe) [inline libs/designsystem/action-group/src/action-group.component.ts]: Remove 🔥 
```suggestion
  imports: [CommonModule, ButtonComponent, ItemModule, MenuComponent],
```

File: libs/designsystem/menu/src/menu.component.ts [hunk 3]
```diff
@@ -76,12 +86,23 @@ export class MenuComponent implements AfterViewInit {
...
+        this.floatingDirective.hide();
+      });
+    });
   }
 
-  @HostListener('document:ionScroll')
-  _onIonScroll() {
-    this.floatingDirective.hide();
+  ngOnDestroy(): void {
+    this.scrollListenerDisposeFn();
```
Comment (jakobe) [inline libs/designsystem/menu/src/menu.component.ts]: Should we null check just for good measure? 🤔

File: libs/designsystem/menu/src/menu.component.ts [hunk 4]
```diff
@@ -103,6 +103,8 @@ export class MenuComponent implements AfterViewInit, OnDestroy {
   }
 
   ngOnDestroy(): void {
-    this.scrollListenerDisposeFn();
+    if (this.scrollListenerDisposeFn() != null) {
+      this.scrollListenerDisposeFn();
+    }
```
Comment (jakobe) [inline libs/designsystem/menu/src/menu.component.ts]: Or simply:
```suggestion
    this.scrollListenerDisposeFn?.();
```

**Review by jakobe — DISMISSED** [review]

File: libs/designsystem/menu/src/menu.component.ts (hunk 4 above)
Comment (RasmusKjeldgaard) [reply libs/designsystem/menu/src/menu.component.ts]: Neat!

**Review by jakobe — APPROVED** [review]
Good job! 👏🏻 
:shipit:

**Pull request merged**